README.markdown

```diff
@@ -467,20 +469,23 @@ let value = max(x, y, z)  // another free function that feels natural
 
 Use trailing closure syntax only if there's a single closure expression parameter at the end of the argument list. Give the closure parameters descriptive names.
 
-For method calls with multiple closures, use a descriptive name for each closure parameter, and separate each closure parameter out onto a new line.
+For method calls with multiple closures, use a descriptive name for each closure parameter, and add a newline before each parameter name (including the first)
```

Comment:
Maybe "for method calls where a closure is specified by name (rather than using the trailing closure syntax) ..."